src/modules/control_allocator/ActuatorEffectiveness/ActuatorEffectivenessStandardVTOL.cpp

```diff
@@ -83,6 +84,27 @@ void ActuatorEffectivenessStandardVTOL::allocateAuxilaryControls(const float dt,
 	}
 }
 
+void ActuatorEffectivenessStandardVTOL::updateSetpoint(const matrix::Vector<float, NUM_AXES> &control_sp,
+		int matrix_index, ActuatorVector &actuator_sp, const matrix::Vector<float, NUM_ACTUATORS> &actuator_min,
+		const matrix::Vector<float, NUM_ACTUATORS> &actuator_max)
+{
+	if (matrix_index == 0) {
+		// Stop front facing motors when they are commanded 0 thrust
```

Comment:
Don't we just want to keep them at PWM_MIN? Stop for me is to, well, stop completely, which is what most VTOL pusher/pullers should be confiugred to do, but if one wants to have them always spinning that should be possible by increasing PWM_MIN to higher than 1000 right?

Reply:
> which is what most VTOL pusher/pullers should be confiugred to do

That's what I'm trying to solve that you have to configure PWM according to what the motor is supposed to do which is context the allocator has but you currently need to set manually.

> if one wants to have them always spinning that should be possible by increasing PWM_MIN to higher than 1000 right?

It's possible and currently, every user has to manually set the PWM_MIN of any motor that should not stop spinning e.g. any multicopter motor to a higher value. Also consider this only works for PWM outputs and the special cases all have to be manually configured for any other interface. E.g. DSHOT pusher is not even possible because the functionality is captured manually in PWM configuration. Instead of re-adding some magic to guess which motor should not stop spinning without having the context I suggest configuring every motor to be the same way and the allocator to have the interface available to command a motor to stop depending on what it does. So in summary it's extending the VTOL motor start/stop interface to all cases a motor should start/stop.